apps/ledger-live-desktop/src/newArch/features/MemoTag/components/MemoTagField.tsx

```diff
@@ -44,8 +45,20 @@ const MemoTagField = ({
   placeholder,
   label,
   tooltipText,
+  validationHandler,
 }: MemoTagFieldProps) => {
   const { t } = useTranslation();
+  const [memoValue, setMemoValue] = useState(value);
+  const debouncedMemoValue = useDebounce(memoValue, 300);
+
+  useEffect(() => {
+    if (debouncedMemoValue !== value) onChange?.(debouncedMemoValue || "");
```

Comment:
when the coin is number based memo field, if the `debouncedMemo` value equals `0` , it will force it to empty.

Reply:
Thanks you're right, I will check and fix it

Reply:
As we receive a string we're good.
<img width="333" alt="image" src="https://github.com/user-attachments/assets/dc4351a6-9fb2-4ac4-8426-ba4b7bdfcfe3">

Reply:
perfect